voucher/urls.py

```diff
@@ -3,8 +3,11 @@
 
 # SharedAPIRootRouter is automatically imported in global urls config
 router = SharedAPIRootRouter()
-router.register(r'voucher/wallets', WalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/voucher', VoucherWalletViewSet, base_name='voucher_wallets')
+router.register(r'voucher/wallets/coffee', CoffeeWalletViewSet, base_name='coffee_wallets')
+router.register(r'voucher/registerlogs/voucher', WorkLogViewSet)
+router.register(r'voucher/registerlogs/coffee', RegisterLogViewSet)
+router.register(r'voucher/uselogs/voucher', VoucherUseLogViewSet)
+router.register(r'voucher/uselogs/coffee', CoffeeUseLogViewSet)
```

Comment:
Man har jo også det problemet at man bruker timer for bong walletene, så da har du mere sjekking man må igjennom :/